backend/courses/management/commands/load_status_history.py

```diff
@@ -94,7 +94,5 @@ def handle(self, *args, **kwargs):
 
             print(f"Finished loading status history from {src}... processed {row_count} rows. ")
 
-            print(
-                f"Recomputing PCA Demand Distribution Estimates for {len(semesters)} semesters..."
-            )
-            recompute_percent_open(semesters=",".join(semesters), verbose=True)
+            print(f"Recomputing PCA Stats for {len(semesters)} semesters...")
+            recompute_stats(semesters=",".join(semesters), verbose=True)
```

Comment:
How often is this called? Is there a possible edge case where the fields added to the DB by `recompute_stats` and added in this PR are not actually present when the user first goes to add, for instance, a day time filter?

Reply:
Actually looking at the migration, I think this is accounted for

Reply:
Yeah recompute_stats is run right after the registrar import (run in a cron job every ~24 hours), and I believe the registrar import is the only place that would add/remove sections or meetings.